src/dev/ci_setup/setup_env.sh

```diff
@@ -175,4 +179,15 @@ if [[ -d "$ES_DIR" && -f "$ES_JAVA_PROP_PATH" ]]; then
   export JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
 fi
 
+###
+### copy .bazelrc-ci into $HOME/.bazelrc
+###
+cp -f "$KIBANA_DIR/src/dev/ci_setup/.bazelrc-ci" "$HOME/.bazelrc";
+
+###
+### append auth token to buildbuddy into "$HOME/.bazelrc";
+###
+echo "# Appended by $KIBANA_DIR/src/dev/ci_setup/setup.sh" >> "$HOME/.bazelrc"
+echo "build --remote_header=x-buildbuddy-api-key=$KIBANA_BUILDBUDDY_CI_API_KEY" >> "$HOME/.bazelrc"
```

Comment:
Doesn't look like you're still setting `KIBANA_BUILDBUDDY_CI_API_KEY`

Reply:
We are, same settings used on https://github.com/elastic/kibana/pull/90116 are reporting on build buddy. They are not here because we haven't yet merged the code to setup the variable, its on the other PR

Reply:
Ah, that's right.